src/lighting/lighting-controller.js

```diff
@@ -368,6 +372,9 @@ class LightingController {
         _self.request.time = moment().add(5, 'minutes');
         _self.turnOffLights();
         break;
+      case 'color':
```

Comment:
Correct, it's not necessary and most likely would not be used.  I'll remove it and can add it back if there's ever a need.